src/authorize.js

```diff
@@ -95,35 +104,28 @@ async function authorize(token, params, referer) {
 
   const id = params.id.split("/").slice(-1)[0];
 
-  const currentUser = getCurrentUser(token);
-  const doc = await getDoc(id);
-  const visibility = getVisibility(doc._source);
-  const username = isObject(currentUser) ? currentUser.displayName[0] : 'Anonymous';
-
-  console.log('User:', username);
-  console.log('Item ID:', id);
-  console.log('Visibility:', visibility);
+  return await getImageAuthorization(id);
+}
 
-  switch(visibility) {
-    case 'open':          return true;
-    case 'authenticated': return isObject(currentUser);
-    case 'restricted':    return false;
-    default:              return true;
-  }
+async function getImageAuthorization(id) {
+  const response = await axios.get(
```

Comment:
This project doesn't use `axios`; it uses `node-fetch`. I realize we're kind of split on that in our repos, and since DC API uses `axios` for all of its regular HTTP/S requests, maybe we should switch this project over as well. I'll leave it up to you whether to convert all of the fetch code to axios code, or just require fetch up at the top and change this to 
```suggestion
  const response = await fetch(
```
(in which case you need to change the check to `response.status` instead of `response.statusCode`)

If you decide to switch over, there's some fetch code in [`prepare_template.js`](https://github.com/nulib/iiif-server-terraform/blob/3166-use-dc-api-v2-auth/scripts/prepare_template.js#L12) that'll have to get changed as well. Or you could leave both dependencies, with `fetch` used in the deploy script and `axios` in the lambda itself. I'm flexible on what gets changed now and what gets cleaned up later.